### DISCUSSION TITLE
Unable to Run code Successfully on Visual Studio

### DISCUSSION THREAD

**Discussion by chokostica** [community/community#66379] [Discussions]
### Select Topic Area

Question

### Body

[csma-cd-protocol-main.zip](https://github.com/community/community/files/12551388/csma-cd-protocol-main.zip)

I cloned the attached repository on Visual Studio C++. It appears there is no .sln file and hence the code does not run. Someone help with steps how I can run this code successfully on C++.

**Comment by Akash1134** [comment]
Closing as a duplicate of #66381

**Comment by Akash1134** [comment]
Hi @udlondy ,

The GitHub Community on Discussions is a space for all GitHub users to come and discuss ideas and projects. Your post was removed because it contained content that violates our Community Code of Conduct, including posting the same comment in multiple places. As a result, we have hidden the duplicate comments, and we ask that you no longer cross-post.

For additional guidance, please review our [Community on Discussions Code of Conduct](https://github.com/community/community/blob/main/CODE_OF_CONDUCT.md) and [GitHub’s Terms of Service](https://docs.github.com/en/github/site-policy/github-terms-of-service).
